lib/prompt.js

```diff
@@ -120,7 +120,18 @@ class Prompt extends Events {
       await this.options.onSubmit.call(this, this.name, this.value, this);
     }
 
-    let result = this.state.error || await this.validate(this.value, this.state);
+    let result;
+    try {
+      result =
+        this.state.error || (await this.validate(this.value, this.state));
```

Comment:
please format this to be on one line. thanks!

Reply:
Done